typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aiconfig",
-  "version": "1.1.7",
```

Comment:
was 1.1.8 already published?

Reply:
No, but I just did this to be consistent with last week's "release"